modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/CustomMustacheFactory.java

```diff
@@ -360,4 +360,28 @@ public void encode(String s, Writer writer) throws IOException {
             writer.write(URLEncoder.encode(s, StandardCharsets.UTF_8));
         }
     }
+
+    /**
+     * Build a new {@link CustomMustacheFactory} object.
+     */
+    static class Builder {
+        private String mediaType = DEFAULT_MEDIA_TYPE;
+        private boolean detectMissingParams = false;
```

Comment:
Nitpick: Please create a static final boolean `DEFAULT_DETECT_MISSING_PARAMS` and reference it here for readability.

Reply:
Done. Thank you for the feedback.